### DISCUSSION TITLE
Inquiry

### DISCUSSION THREAD

**Discussion by drfeinberg** [ofek/pyapp#58] [General]
Hey  this looks interesting. How does it compare to PyInstaller? Can it compile things with hidden imports?

**Comment by ofek** [comment]
That is one of the main differences, here we do not do any build sorcery. In the end applications run using a real Python installation on disk. This also means there is no chance that certain extension modules do not work randomly.

**Reply by drfeinberg** [reply]
Cool.  I'll check it out.  Thanks!

**Reply by ofek** [reply]
Please let me know how your experience goes!
